modules/nf-core/backsub/tests/main.nf.test.snap

```diff
@@ -0,0 +1,15 @@
+{
+    "markerout": {
+        "content": [
+            [
+                [
+                    {
+                        "id": "test"
+                    },
+                    "test.csv:md5,28fddf3b89ed0a9b17b1f8f28e754612"
+                ]
+            ]
```

Comment:
im not seeing the versions in the snapshot

Reply:
Thanks for the catch!